profiling/src/pprof/sliced_proto.rs

```diff
@@ -85,12 +85,6 @@ pub struct ProfileFunctionsEntry {
     pub functions_entry: Function,
 }
 
-#[derive(Eq, Hash, PartialEq, ::prost::Message)]
-pub struct ProfileStringTableEntry {
-    #[prost(string, required, tag = "6")]
-    pub string_table_entry: String,
-}
-
```

Comment:
Why the change to do this differently from all the other field serializers?

Reply:
Because `prost::Message` requires a `String` to encode strings. You can't encode any other `String`-like representations such as `&str`, `Box<str>`, etc with a `prost::Message`.